### ISSUE TITLE
`clusteripprefixset` not injected into local RIB

### ISSUE THREAD

**Issue by tamihiro** [cloudnativelabs/kube-router#772]
Upon trying to test ebgp ecmp to External IP, I found that External IP would never made it to kube-router's local RIB, and therefore it wasn't advertised to ebgp peers. `gobgp policy` command shows the following.

```Name kube_router_import:
    StatementName kube_router_import_stmt0:
      Conditions:
        PrefixSet: any clusteripprefixset
      Actions:
         reject
```
So GoBGP would apply this policy, and reject to import not only paths received from all peers but also those local injected by the controller. 

I have submitted PR #771 for a fix.

**Comment by tamihiro** [comment]
Sorry I had overkooked #767 until after shooting my PR.
@aauren I'm afraid I'm quite unclear why you reverted #721 which you had submitted yourself.

With [the new import policy](https://github.com/cloudnativelabs/kube-router/pull/721#issuecomment-496019375) since #721 was merged, GoBGP rejects to locally install the path of service VIPs requested by the controller's `nrc.bgpServer.AddPath()` method. Consequently, the route to service VIPs is never advertised to external BGP peers. I believe `kube_router_import_stmt0` is meant to only reject updates of service VIPs from any other BGP speakers, including kube-routers on other nodes.

If this is the issue you're addressing, I don't think you need to revert your previous PR. My PR will add just some more code to `bgp_policies.go` so `kube_router_import_stmt0` can include a NeighborSet condition.
```Name kube_router_import:
    StatementName kube_router_import_stmt0:
      Conditions:
        PrefixSet: any clusteripprefixset
        NeighborSet: any allpeerset
      Actions:
         reject
```

Please correct me if I'm missing something as I'm quite new to kube-router (and the whole k8s thing).

**Comment by aauren** [comment]
No worries, I'm pretty new around here too.

No, I think that your approach is right. To be honest, I just hadn't considered adding an additional match to the policy. I filed the revert because since my PR had been merged the master doesn't work correctly and it was causing people to file multiple issues. I was planning on following it up with replacement logic in the route controller code that did essentially the same thing as the import policy.

I saw your PR yesterday morning, but didn't get the chance to try it out yesterday like I was hoping.

If it checks out then I think this would be better than reverting my PR and adding additional logic to the route controller code.

**Comment by murali-reddy** [comment]
thanks for the PR and figuring out the issue @tamihiro 

>If it checks out then I think this would be better than reverting my PR and adding additional logic to the route controller code.

Yes, that would be the right thing. I was thinking to fix than to revert, thankfully we have a PR now.

Will test out #771

**Comment by aauren** [comment]
@murali-reddy I can confirm that this fixes the problem caused by #721 where advertised routes were disappearing at the first sync interval.

I would recommend that we merge #771

**Comment by andrewsykim** [comment]
Looks like this was fixed
